as.c

```diff
@@ -186,12 +197,15 @@ static inline vm_operand make_operand(vm_env *env, char *line, const char *data)
         op.value.id = vm_add_const(env, STR, quoted_strdup(data));
         break;
     default:
-        printf("Error: please specify operand type for '%s' in the following line\n"
-               "       %s\n\n"
-               "Supported types:\n"
-               "       $ (constant integer)\n"
-               "       # (temp integer)\n"
-               "       ^ (constant string\n", data, line);
+        printf(
+            "Error: please specify operand type for '%s' in the following "
+            "line\n"
+            "       %s\n\n"
+            "Supported types:\n"
+            "       $ (constant integer)\n"
+            "       # (temp integer)\n"
+            "       ^ (constant string\n",
```

Comment:
This line is out-of-date for string literals. Please minimize the valid changes.